manifests/s/spyder/spyder/5.1.1/spyder.spyder.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: spyder.spyder
+PackageVersion: 5.1.1
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/spyder-ide/spyder/releases/latest/download/Spyder_64bit_full.exe

```

```suggestion
  InstallerUrl: https://github.com/spyder-ide/spyder/releases/download/v5.1.1/Spyder_64bit_full.exe
```
Since wingetbot might miss updating this application, it's probably worth to change it to the version specific one.